src/test/java/org/broadinstitute/hellbender/utils/SequenceDictionaryUtilsUnitTest.java

```diff
@@ -115,6 +114,12 @@
                 // These have isReadsToReferenceComparison == false, so we don't expect an exception:
                 { Arrays.asList(CHRM_HG19, CHR1_HG19),                              Arrays.asList(CHR1_HG19),                       DIFFERENT_INDICES, null, false, hg19AllContigsIntervalSet },
                 { Arrays.asList(CHR1_HG19, CHR_NONSTANDARD1, CHR2_HG19, CHRM_HG19), Arrays.asList(CHR1_HG19, CHR2_HG19, CHRM_HG19), DIFFERENT_INDICES, null, false, hg19AllContigsIntervalSet },
+
+                // tests for SUPERSETS
+
+                { Arrays.asList(CHR1_HG19, CHR2_HG19),                                  Arrays.asList(CHR1_HG19),                                                       SUPERSETS, null, true, hg19AllContigsIntervalSet },
+                { Arrays.asList(CHR1_HG19, CHR_NONSTANDARD1),                           Arrays.asList(CHR1_HG19),                                                       SUPERSETS, null, true, null },
+                { Arrays.asList(CHR1_HG19, CHR2_HG19, CHR10_HG19, CHR_NONSTANDARD1),    Arrays.asList(CHR1_HG19, CHR2_HG19, CHR10_HG19),                                SUPERSETS, null, false, null },
```

Comment:
Branch is not compiling because you didn't change `SUPERSETS` -> `SUPERSET` here